HelpMyStreetFE/HelpMyStreetFE/src/js/authenticate.js

```diff
@@ -0,0 +1,23 @@
+import { processYoti } from "./profile/yoti";
+
+var getParameterByName = function (name, url) {
+    if (!url) url = window.location.href;
+    name = name.replace(/[\[\]]/g, '\\$&');
+    var regex = new RegExp('[?&]' + name + '(=([^&#]*)|&|#|$)'),
+        results = regex.exec(url);
+    if (!results) return null;
+    if (!results[2]) return '';
+    return decodeURIComponent(results[2].replace(/\+/g, ' '));
+}
```

Comment:
https://www.sitepoint.com/get-url-parameters-with-javascript/

Not supported in IE11 but Babel should polyfill I think 🤞

Reply:
this code had been used in live before so i reckon were good here

Reply:
just noticed this is now defined in at least two places, could put a single definition in to shared somewhere and import it to reduce code duplication